src/bindings/python/wheel/CMakeLists.txt

```diff
@@ -18,10 +18,12 @@ if(LINUX)
                       NAMES patchelf
                       DOC "Path to patchelf tool")
     if(NOT patchelf_program)
-        message(FATAL_ERROR "patchelf is not found, which is needed to build ie_wheel")
+        message(FATAL_ERROR "patchelf is not found. It is required to build OpenVINO Runtime wheel")
     endif()
 endif()
 
+ie_check_pip_package(wheel FATAL_ERROR)
```

Comment:
1. should we also check `setuptools>=53.0.0`?
2. why don't we check exact version of `wheel`. I see we have `wheel>=0.36.2` in `requirements.txt`
3. should we check the whole file like `ie_check_pip_package(${CMAKE_CURRENT_SOURCE_DIR}/requirements-dev.txt FATAL_ERROR)`

Reply:
added ov_check_pip_packages function to check entire ${CMAKE_SOURCE_DIR}/src/bindings/python/wheel/requirements-dev.txt
[cmake/developer_package/IEDevScriptsConfig.cmake](https://github.com/openvinotoolkit/openvino/pull/11278/files#diff-bebb96ad9d754ddecb77341eb85eeb4d4736a623b1dfb8cdf6a9228545462fbc)